src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatabilityAnalyzerTests.cs

```diff
@@ -180,6 +180,46 @@ public int M3(int option)
             await VerifyAnalyzerAsyncCs(source);
         }
 
+        [Fact]
+        public async Task OsDependentPropertyGetterSetterCalledWarns()
+        {
+            var source = @"
+using System.Runtime.Versioning;
+
+public class Test
+{
+    public static bool WindowsOnlyPropertyGetter
+    {
+        [SupportedOSPlatform(""windows"")]
+        get { return true; }
+        set { }
+    }
+
+    public static bool WindowsOnlyPropertySetter
+    {
+        get { return true; }
+        [SupportedOSPlatform(""windows"")]
+        set { }
+    }
+
+    public void M1()
+    {
+        WindowsOnlyPropertyGetter = true;
```

Comment:
Can you please add a test where the reference is a read + write, i.e. say `WindowsOnlyPropertyGetter += value;` and `WindowsOnlyPropertySetter += value;`?

Reply:
Good point

Reply:
Can you please also add a test where you pass property as ref, in, out argument to a method call?

Reply:
> Can you please also add tests where you pass property as ref, in, out argument to a method call?

I tried, got compiler error CS0206: `A property or indexer may not be passed as an out or ref parameter, so i am not handling the case ValueUsageInfo.Reference at all

Reply:
Ah, thanks! Would you be able to add comment on the same lines in this method?

Reply:
I mean a comment in your analyzer where it processes value usage info.